.github/workflows/integration-auth-tests.yml

```diff
@@ -84,6 +84,8 @@ jobs:
           yq eval '.server.auth.provider_config.jwks.token = "${{ env.TOKEN }}"' -i $run_dir/run.yaml
           cat $run_dir/run.yaml
 
+          # avoid line breaks in the server log, especially because we grep it below.
+          export COLUMNS=1984
```

Comment:
thanks, that does fix the current test, but I wonder whether it wouldn't make more sense to make the test assertions more robust to don't make exact string matches but match only on the parts that are important, to avoid breaking on formatting or rewording changes ? Messages aren't stable APIs.

Reply:
While I agree with the approach in general, I'm not sure how we could relax that `grep` line to do a `contains` versus a strict match:

```
grep -q "Enabling authentication with provider: ${{ matrix.auth-provider }}" server.log
```